madge/src/main/java/com/jakewharton/madge/MadgeCanvas.java

```diff
@@ -22,27 +24,25 @@
 final class MadgeCanvas extends DelegateCanvas {
   private static final int DEFAULT_COLOR = 0x88FF0088;
   private static final int TEXT_SIZE_DP = 14;
+  private static final int GRID_SIZE_PX = 512;
```

Comment:
Couldn't it be really small if we draw the grid with a bitmap shader with [tile mode configured](https://github.com/android/platform_frameworks_base/blob/android-5.0.0_r7/graphics/java/android/graphics/drawable/BitmapDrawable.java#L501-L503)?